server/index.js

```diff
@@ -209,6 +257,36 @@ app.post('/api/orders', (req, res, next) => {
     });
 });
 
+app.post('/send', (req, res, next) => {
+  const email = req.body.email;
+  const orderId = req.body.orderId;
+  const content = req.body.html;
+
+  createEmail(content)
+    .then(emailHTML => {
+
+      const mail = {
+        from: process.env.EMAIL_USER,
+        to: email,
+        subject: `Thank you for shopping at BlytheJubilee, Order #${orderId}`,
+        html: emailHTML
+      };
+
+      sendEmail(mail, (err, data) => {
+        if (err) {
```

Comment:
You should always pass errors to `console.error()` at the very least so you know _why_ something failed. In this case, you can pass it to `next()`.